common/djangoapps/philu_commands/management/commands/send_referral_follow_up_email.py

```diff
@@ -0,0 +1,41 @@
+from datetime import timedelta
+from django.utils import timezone
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from openedx.features.smart_referral.models import SmartReferral
+from openedx.features.smart_referral.tasks import task_send_referral_follow_up_email
+
+CONTACT_EMAIL = 'contact_email'
+DAYS_TO_SEND_FOLLOW_UP_EMAIL = 3
+IS_REFERRAL_STEPS_COMPLETED = 'is_referral_step_complete'
+
+
+class Command(BaseCommand):
+    help = """
+    Send follow up referral email to all those contacts
+    """
+
+    def handle(self, *args, **options):
+        emails_list = []
+        three_days_old_date = timezone.now().date() - timedelta(days=DAYS_TO_SEND_FOLLOW_UP_EMAIL)
+
+        three_days_old_referral = SmartReferral.objects.filter(
```

Comment:
It can be renamed to `three_days_old_pending_referrals` to give more context

Reply:
I think 'three_days_old_referral' name is also meaningful and explains its purpose.

Reply:
I agree, Its fine as well.. 
It would be great if you make it plural, so that it shows that it would have multiple referrals